packages/theme-docs/src/layouts/default.vue

```diff
@@ -28,6 +28,7 @@ export default {
   },
   head () {
     const i18nSeo = this.$nuxtI18nSeo()
+    const previewUrl = this.settings.url.replace(/\/$/, '') + '/preview.png'
```

Comment:
This is implicitly making it mandatory to have `preview.png` in `static` dir without warning

Reply:
Indeed, it was already the case, I think we should improve this part: https://content.nuxtjs.org/themes/docs#static

Reply:
Indeed but maybe we can also check it by module and show a warning or simply avoid adding previwUrl if file doesn't exists?